### PR TITLE
Language constants: Consolidate defines for Product Model

### LINKED ISSUE #603: Language define consolidation - Model Name
If you look at a string like Model, it's defined 13 times in the catalog and 33 times in the admin.  Now some of these are different from others, but if you just consolidated (say) TEXT_PRODUCTS_MODEL, you'd get a dozen of them.  What if we promote these to the includes/languages/english file in the catalog and admin?

### PR DESCRIPTION
**Pull request by scottcwilson**
Consolidates model and title string for model to one place in catalog, one place in admin, reducing by 42 the number of strings that need to be updated if this name is to be changed.  PRODUCTS_NAME, _DESCRIPTION and _QUANTITY are other strings which could be consolidated in the same way if desired, but MODEL seems like the best choice, since some stores have a strong preference for another name here.  Fixes #603

### CODE REVIEW

**Comment by zcwilt** [comment]
Where appropriate, can you update the copyright dates.
Thanks

File: admin/includes/languages/english/products_to_categories.php [hunk 1]
```diff
@@ -14,13 +14,11 @@
 
 define('TABLE_HEADING_PRODUCTS_ID', 'Prod ID');
 define('TABLE_HEADING_PRODUCT', 'Product Name');
-define('TABLE_HEADING_MODEL', 'Model');
```
Comment (drbyte) [inline admin/includes/languages/english/products_to_categories.php]: You've removed this, but not re-defined it elsewhere, yet it's used in `category_product_listing.php` and `downloads_manager.php`

**Comment by zcwilt** [comment]
:+1:

**Comment by drbyte** [comment]
:+1:

**Pull request merged**